src/transformers/models/llava/processing_llava.py

```diff
@@ -45,6 +48,10 @@ class LlavaProcessor(ProcessorMixin):
 
     def __init__(self, image_processor=None, tokenizer=None):
         super().__init__(image_processor, tokenizer)
+        self.image_size = self.image_processor.size["shortest_edge"]
+        self.patch_size = getattr(self.image_processor, "patch_size", None)
+        self.image_token = "<image>"
```

Comment:
Is this consistent for all the tokenizers?

Reply:
For tokenizers, yes! The only part from tokenizer is "image token" which I believe is established as `"<image>"` for LLaVas.

For all image processors/vision towers, then maybe no. Calculation of image token seq length is done assuming we get CLIP as image tower. I am not sure if other vision backbones are all similar or require different handling. 

In case there are llavas w/o CLIP and it has to be handled differently, I will see if we can unify it without extra code-hacks

Reply:
For existing possible backbones (CLIP, SigLIP, Dino, ViT) the current code will work as expected. Same for tokenizers, and in case any model decided to change image token, the user can set it by `processr.image_token="{ANY_NEW_TOKEN}"`